lib/std/os/linux/bpf.zig

```diff
@@ -887,7 +887,7 @@ pub const Cmd = enum(usize) {
     map_get_next_key,
 
     /// Verify and load an eBPF program, returning a new file descriptor
-    /// associated with  the  program.   The close-on-exec file descriptor flag
+    /// associated with  the  program.  The close-on-exec file descriptor flag
```

Comment:
the spacing is still weird here.

Reply:
true, I removed the double spaces between words as well and force pushed.

Reply:
There's still 2 spaces before the `The`.

Reply:
Yes, I left that because that was how the rest of the file was formatted. But I can see that the rest of the comments in the project seem to use single space after period, so I'll reformat this file to align with that.